Fix Dockerfile for 1.8

I have run a test and it seems to fail.
```
$ git diff .
diff --git i/docker/Dockerfile w/docker/Dockerfile
index 4a836dc..a174cc6 100644
--- i/docker/Dockerfile
+++ w/docker/Dockerfile
@@ -10,7 +10,7 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash && \
 
 # Install mwoffliner
 RUN apt-get install -y nscd file imagemagick && \
-    npm install -g mwoffliner && \
+    npm install -g openzim/mwoffliner && \
     apt-get remove -y make g++
 
 # Boot commands
```

and I get
```
Sending build context to Docker daemon  5.632kB
Step 1/5 : FROM openzim/zimwriterfs:latest
 ---> 9cb355d0740b
Step 2/5 : RUN apt-get update && apt-get install -y make g++ curl git
 ---> Running in a4e36fe900dc
Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:5 http://security.ubuntu.com/ubuntu bionic-security/universe Sources [40.2 kB]
Get:6 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [344 kB]
Get:7 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [3,440 B]
Get:8 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [156 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic/universe Sources [11.5 MB]
^C
kelson@camber:~/code/mwoffliner/docker$ emacs -nw Dockerfile 

[3]+  Stopped                 emacs -nw Dockerfile
kelson@camber:~/code/mwoffliner/docker$ docker build .
Sending build context to Docker daemon  5.632kB
Step 1/5 : FROM openzim/zimwriterfs:latest
 ---> 9cb355d0740b
Step 2/5 : RUN apt-get update && apt-get install -y make g++ curl git
 ---> Running in 81d74267fec5
Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:5 http://security.ubuntu.com/ubuntu bionic-security/universe Sources [40.2 kB]
Get:6 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [3,440 B]
Get:7 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [344 kB]
Get:8 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [156 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic/universe Sources [11.5 MB]
Get:10 http://archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages [13.5 kB]
Get:11 http://archive.ubuntu.com/ubuntu bionic/multiverse amd64 Packages [186 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages [11.3 MB]
Get:13 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages [1,344 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic-updates/universe Sources [182 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [10.8 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [941 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [685 kB]
Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [6,963 B]
Get:19 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [3,650 B]
Fetched 27.3 MB in 4s (7,239 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  autotools-dev dirmngr fakeroot gir1.2-glib-2.0 gir1.2-harfbuzz-0.0 gnupg
  gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf
  gpgsm icu-devtools libalgorithm-diff-perl libalgorithm-diff-xs-perl
  libalgorithm-merge-perl libassuan0 libdpkg-perl libelf1 libfakeroot
  libfile-fcntllock-perl libfreetype6 libgirepository-1.0-1 libglib2.0-0
  libglib2.0-bin libglib2.0-data libglib2.0-dev-bin libgraphite2-3
  libgraphite2-dev libharfbuzz-gobject0 libharfbuzz-icu0 libharfbuzz0b
  libicu-le-hb0 libicu60 libiculx60 libksba8 liblocale-gettext-perl
  libltdl-dev libltdl7 libnpth0 libpcre16-3 libpcre3-dev libpcre32-3
  libpcrecpp0v5 libpng16-16 libsigsegv2 libxml2 m4 pinentry-curses
  python3-distutils python3-lib2to3 shared-mime-info xdg-user-dirs
Use 'apt autoremove' to remove them.
The following additional packages will be installed:
  libcurl4
Suggested packages:
  g++-multilib gettext-base git-daemon-run | git-daemon-sysvinit git-doc
  git-el git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn make-doc
The following NEW packages will be installed:
  curl g++ git libcurl4 make
0 upgraded, 5 newly installed, 0 to remove and 66 not upgraded.
Need to get 4,436 kB of archives.
After this operation, 33.6 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcurl4 amd64 7.58.0-2ubuntu3.6 [214 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 curl amd64 7.58.0-2ubuntu3.6 [159 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 g++ amd64 4:7.3.0-3ubuntu2.1 [1,572 B]
Get:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 git amd64 1:2.17.1-1ubuntu0.4 [3,907 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic/main amd64 make amd64 4.1-9.1ubuntu1 [154 kB]
Fetched 4,436 kB in 0s (14.1 MB/s)
Selecting previously unselected package libcurl4:amd64.
(Reading database ... 15529 files and directories currently installed.)
Preparing to unpack .../libcurl4_7.58.0-2ubuntu3.6_amd64.deb ...
Unpacking libcurl4:amd64 (7.58.0-2ubuntu3.6) ...
Selecting previously unselected package curl.
Preparing to unpack .../curl_7.58.0-2ubuntu3.6_amd64.deb ...
Unpacking curl (7.58.0-2ubuntu3.6) ...
Selecting previously unselected package g++.
Preparing to unpack .../g++_4%3a7.3.0-3ubuntu2.1_amd64.deb ...
Unpacking g++ (4:7.3.0-3ubuntu2.1) ...
Selecting previously unselected package git.
Preparing to unpack .../git_1%3a2.17.1-1ubuntu0.4_amd64.deb ...
Unpacking git (1:2.17.1-1ubuntu0.4) ...
Selecting previously unselected package make.
Preparing to unpack .../make_4.1-9.1ubuntu1_amd64.deb ...
Unpacking make (4.1-9.1ubuntu1) ...
Setting up make (4.1-9.1ubuntu1) ...
Setting up libcurl4:amd64 (7.58.0-2ubuntu3.6) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Setting up g++ (4:7.3.0-3ubuntu2.1) ...
update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/c++.1.gz because associated file /usr/share/man/man1/g++.1.gz (of link group c++) doesn't exist
Setting up git (1:2.17.1-1ubuntu0.4) ...
Setting up curl (7.58.0-2ubuntu3.6) ...
Removing intermediate container 81d74267fec5
 ---> ba4132c39619
Step 3/5 : RUN curl -sL https://deb.nodesource.com/setup_10.x | bash &&     apt-get install --yes nodejs
 ---> Running in 4bca8d40ae19

## Installing the NodeSource Node.js 10.x repo...


## Populating apt-get cache...

+ apt-get update
Hit:1 http://security.ubuntu.com/ubuntu bionic-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu bionic InRelease
Hit:3 http://archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease
Reading package lists...

## Installing packages required for setup: lsb-release...

+ apt-get install -y lsb-release > /dev/null 2>&1

## Confirming "bionic" is supported...

+ curl -sLf -o /dev/null 'https://deb.nodesource.com/node_10.x/dists/bionic/Release'

## Adding the NodeSource signing key to your keyring...

+ curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
Warning: apt-key output should not be parsed (stdout is not a terminal)
OK

## Creating apt sources list file for the NodeSource Node.js 10.x repo...

+ echo 'deb https://deb.nodesource.com/node_10.x bionic main' > /etc/apt/sources.list.d/nodesource.list
+ echo 'deb-src https://deb.nodesource.com/node_10.x bionic main' >> /etc/apt/sources.list.d/nodesource.list

## Running `apt-get update` for you...

+ apt-get update
Hit:1 http://archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://security.ubuntu.com/ubuntu bionic-security InRelease
Hit:3 http://archive.ubuntu.com/ubuntu bionic-updates InRelease
Get:4 https://deb.nodesource.com/node_10.x bionic InRelease [4,584 B]
Hit:5 http://archive.ubuntu.com/ubuntu bionic-backports InRelease
Get:6 https://deb.nodesource.com/node_10.x bionic/main amd64 Packages [763 B]
Fetched 5,347 B in 2s (2,837 B/s)
Reading package lists...

## Run `sudo apt-get install -y nodejs` to install Node.js 10.x and npm
## You may also need development tools to build native addons:
     sudo apt-get install gcc g++ make
## To install the Yarn package manager, run:
     curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
     sudo apt-get update && sudo apt-get install yarn


Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  autotools-dev dirmngr fakeroot gir1.2-glib-2.0 gir1.2-harfbuzz-0.0 gnupg
  gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf
  gpgsm icu-devtools libalgorithm-diff-perl libalgorithm-diff-xs-perl
  libalgorithm-merge-perl libassuan0 libdpkg-perl libelf1 libfakeroot
  libfile-fcntllock-perl libfreetype6 libgirepository-1.0-1 libglib2.0-0
  libglib2.0-bin libglib2.0-data libglib2.0-dev-bin libgraphite2-3
  libgraphite2-dev libharfbuzz-gobject0 libharfbuzz-icu0 libharfbuzz0b
  libicu-le-hb0 libicu60 libiculx60 libksba8 liblocale-gettext-perl
  libltdl-dev libltdl7 libnpth0 libpcre16-3 libpcre3-dev libpcre32-3
  libpcrecpp0v5 libpng16-16 libsigsegv2 libxml2 m4 pinentry-curses
  python3-distutils python3-lib2to3 shared-mime-info xdg-user-dirs
Use 'apt autoremove' to remove them.
The following additional packages will be installed:
  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python
  python-minimal python2.7 python2.7-minimal
Suggested packages:
  python-doc python-tk python2.7-doc binfmt-support
The following NEW packages will be installed:
  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib nodejs python
  python-minimal python2.7 python2.7-minimal
0 upgraded, 8 newly installed, 0 to remove and 66 not upgraded.
Need to get 19.0 MB of archives.
After this operation, 88.7 MB of additional disk space will be used.
Get:1 https://deb.nodesource.com/node_10.x bionic/main amd64 nodejs amd64 10.15.1-1nodesource1 [15.1 MB]
Get:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.15~rc1-1ubuntu0.1 [334 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.15~rc1-1ubuntu0.1 [1,304 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1 [28.1 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.15~rc1-1ubuntu0.1 [1,912 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.15~rc1-1ubuntu0.1 [238 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1 [7,620 B]
Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1 [140 kB]
Fetched 19.0 MB in 0s (45.9 MB/s)
Selecting previously unselected package libpython2.7-minimal:amd64.
(Reading database ... 16309 files and directories currently installed.)
Preparing to unpack .../0-libpython2.7-minimal_2.7.15~rc1-1ubuntu0.1_amd64.deb ...
Unpacking libpython2.7-minimal:amd64 (2.7.15~rc1-1ubuntu0.1) ...
Selecting previously unselected package python2.7-minimal.
Preparing to unpack .../1-python2.7-minimal_2.7.15~rc1-1ubuntu0.1_amd64.deb ...
Unpacking python2.7-minimal (2.7.15~rc1-1ubuntu0.1) ...
Selecting previously unselected package python-minimal.
Preparing to unpack .../2-python-minimal_2.7.15~rc1-1_amd64.deb ...
Unpacking python-minimal (2.7.15~rc1-1) ...
Selecting previously unselected package libpython2.7-stdlib:amd64.
Preparing to unpack .../3-libpython2.7-stdlib_2.7.15~rc1-1ubuntu0.1_amd64.deb ...
Unpacking libpython2.7-stdlib:amd64 (2.7.15~rc1-1ubuntu0.1) ...
Selecting previously unselected package python2.7.
Preparing to unpack .../4-python2.7_2.7.15~rc1-1ubuntu0.1_amd64.deb ...
Unpacking python2.7 (2.7.15~rc1-1ubuntu0.1) ...
Selecting previously unselected package libpython-stdlib:amd64.
Preparing to unpack .../5-libpython-stdlib_2.7.15~rc1-1_amd64.deb ...
Unpacking libpython-stdlib:amd64 (2.7.15~rc1-1) ...
Setting up libpython2.7-minimal:amd64 (2.7.15~rc1-1ubuntu0.1) ...
Setting up python2.7-minimal (2.7.15~rc1-1ubuntu0.1) ...
Linking and byte-compiling packages for runtime python2.7...
Setting up python-minimal (2.7.15~rc1-1) ...
Selecting previously unselected package python.
(Reading database ... 17057 files and directories currently installed.)
Preparing to unpack .../python_2.7.15~rc1-1_amd64.deb ...
Unpacking python (2.7.15~rc1-1) ...
Selecting previously unselected package nodejs.
Preparing to unpack .../nodejs_10.15.1-1nodesource1_amd64.deb ...
Unpacking nodejs (10.15.1-1nodesource1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Setting up nodejs (10.15.1-1nodesource1) ...
Setting up libpython2.7-stdlib:amd64 (2.7.15~rc1-1ubuntu0.1) ...
Setting up python2.7 (2.7.15~rc1-1ubuntu0.1) ...
Setting up libpython-stdlib:amd64 (2.7.15~rc1-1) ...
Setting up python (2.7.15~rc1-1) ...
Removing intermediate container 4bca8d40ae19
 ---> 67eda843054a
Step 4/5 : RUN apt-get install -y nscd file imagemagick &&     npm install -g openzim/mwoffliner &&     apt-get remove -y make g++
 ---> Running in 7f00f4534e97
Reading package lists...
Building dependency tree...
Reading state information...
file is already the newest version (1:5.32-2ubuntu0.1).
file set to manually installed.
The following packages were automatically installed and are no longer required:
  autotools-dev dirmngr fakeroot gir1.2-glib-2.0 gir1.2-harfbuzz-0.0 gnupg
  gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf
  gpgsm icu-devtools libalgorithm-diff-perl libalgorithm-diff-xs-perl
  libalgorithm-merge-perl libassuan0 libdpkg-perl libelf1 libfakeroot
  libfile-fcntllock-perl libgirepository-1.0-1 libglib2.0-bin
  libglib2.0-dev-bin libgraphite2-dev libharfbuzz-gobject0 libharfbuzz-icu0
  libicu-le-hb0 libiculx60 libksba8 liblocale-gettext-perl libltdl-dev
  libnpth0 libpcre16-3 libpcre3-dev libpcre32-3 libpcrecpp0v5 libsigsegv2 m4
  pinentry-curses python3-distutils python3-lib2to3
Use 'apt autoremove' to remove them.
The following additional packages will be installed:
  dbus fontconfig fontconfig-config fonts-dejavu-core fonts-droid-fallback
  fonts-noto-mono ghostscript gsfonts hicolor-icon-theme imagemagick-6-common
  imagemagick-6.q16 libapparmor1 libavahi-client3 libavahi-common-data
  libavahi-common3 libcairo2 libcap2 libcups2 libcupsfilters1 libcupsimage2
  libdatrie1 libdbus-1-3 libdjvulibre-text libdjvulibre21 libfftw3-double3
  libfontconfig1 libgs9 libgs9-common libidn11 libijs-0.35 libilmbase12
  libjbig0 libjbig2dec0 libjpeg-turbo8 libjpeg8 liblcms2-2 liblqr-1-0
  libmagickcore-6.q16-3 libmagickcore-6.q16-3-extra libmagickwand-6.q16-3
  libnetpbm10 libopenexr22 libpango-1.0-0 libpangocairo-1.0-0
  libpangoft2-1.0-0 libpaper-utils libpaper1 libpixman-1-0 libthai-data
  libthai0 libtiff5 libwmf0.2-7 libxcb-render0 libxcb-shm0 libxrender1 netpbm
  poppler-data ucf
Suggested packages:
  default-dbus-session-bus | dbus-session-bus fonts-noto ghostscript-x
  imagemagick-doc autotrace cups-bsd | lpr | lprng enscript ffmpeg gimp
  gnuplot grads graphviz groff-base hp2xx html2ps libwmf-bin mplayer povray
  radiance sane-utils texlive-base-bin transfig ufraw-batch xdg-utils
  cups-common libfftw3-bin libfftw3-dev liblcms2-utils inkscape libjxr-tools
  libwmf0.2-7-gtk poppler-utils fonts-japanese-mincho | fonts-ipafont-mincho
  fonts-japanese-gothic | fonts-ipafont-gothic fonts-arphic-ukai
  fonts-arphic-uming fonts-nanum
The following NEW packages will be installed:
  dbus fontconfig fontconfig-config fonts-dejavu-core fonts-droid-fallback
  fonts-noto-mono ghostscript gsfonts hicolor-icon-theme imagemagick
  imagemagick-6-common imagemagick-6.q16 libapparmor1 libavahi-client3
  libavahi-common-data libavahi-common3 libcairo2 libcap2 libcups2
  libcupsfilters1 libcupsimage2 libdatrie1 libdbus-1-3 libdjvulibre-text
  libdjvulibre21 libfftw3-double3 libfontconfig1 libgs9 libgs9-common libidn11
  libijs-0.35 libilmbase12 libjbig0 libjbig2dec0 libjpeg-turbo8 libjpeg8
  liblcms2-2 liblqr-1-0 libmagickcore-6.q16-3 libmagickcore-6.q16-3-extra
  libmagickwand-6.q16-3 libnetpbm10 libopenexr22 libpango-1.0-0
  libpangocairo-1.0-0 libpangoft2-1.0-0 libpaper-utils libpaper1 libpixman-1-0
  libthai-data libthai0 libtiff5 libwmf0.2-7 libxcb-render0 libxcb-shm0
  libxrender1 netpbm nscd poppler-data ucf
0 upgraded, 60 newly installed, 0 to remove and 66 not upgraded.
Need to get 23.7 MB of archives.
After this operation, 85.5 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 ucf all 3.0038 [50.5 kB]
Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-dejavu-core all 2.37-1 [1,041 kB]
Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 fontconfig-config all 2.12.6-0ubuntu2 [55.8 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfontconfig1 amd64 2.12.6-0ubuntu2 [137 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic/main amd64 fontconfig amd64 2.12.6-0ubuntu2 [169 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-droid-fallback all 1:6.0.1r16-1.1 [1,805 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libjpeg-turbo8 amd64 1.5.2-0ubuntu5.18.04.1 [110 kB]
Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfftw3-double3 amd64 3.3.7-1 [735 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjbig0 amd64 2.1-3.1build1 [26.7 kB]
Get:10 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjpeg8 amd64 8c-2ubuntu8 [2,194 B]
Get:11 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblcms2-2 amd64 2.9-1ubuntu0.1 [139 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic/main amd64 liblqr-1-0 amd64 0.4.2-2.1 [27.7 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libtiff5 amd64 4.0.9-5ubuntu0.1 [152 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 imagemagick-6-common all 8:6.9.7.4+dfsg-16ubuntu6.4 [60.9 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagickcore-6.q16-3 amd64 8:6.9.7.4+dfsg-16ubuntu6.4 [1,616 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagickwand-6.q16-3 amd64 8:6.9.7.4+dfsg-16ubuntu6.4 [293 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic/main amd64 poppler-data all 0.4.8-2 [1,479 kB]
Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libapparmor1 amd64 2.12-4ubuntu5.1 [31.3 kB]
Get:19 http://archive.ubuntu.com/ubuntu bionic/main amd64 libdbus-1-3 amd64 1.12.2-1ubuntu1 [175 kB]
Get:20 http://archive.ubuntu.com/ubuntu bionic/main amd64 dbus amd64 1.12.2-1ubuntu1 [150 kB]
Get:21 http://archive.ubuntu.com/ubuntu bionic/main amd64 libcap2 amd64 1:2.25-1.2 [13.0 kB]
Get:22 http://archive.ubuntu.com/ubuntu bionic/main amd64 libidn11 amd64 1.33-2.1ubuntu1 [45.7 kB]
Get:23 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-noto-mono all 20171026-2 [75.5 kB]
Get:24 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libavahi-common-data amd64 0.7-3.1ubuntu1.2 [22.1 kB]
Get:25 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libavahi-common3 amd64 0.7-3.1ubuntu1.2 [21.6 kB]
Get:26 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libavahi-client3 amd64 0.7-3.1ubuntu1.2 [25.2 kB]
Get:27 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcups2 amd64 2.2.7-1ubuntu2.3 [211 kB]
Get:28 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcupsimage2 amd64 2.2.7-1ubuntu2.3 [18.6 kB]
Get:29 http://archive.ubuntu.com/ubuntu bionic/main amd64 libijs-0.35 amd64 0.35-13 [15.5 kB]
Get:30 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjbig2dec0 amd64 0.13-6 [55.9 kB]
Get:31 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpaper1 amd64 1.1.24+nmu5ubuntu1 [13.6 kB]
Get:32 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libgs9-common all 9.26~dfsg+0-0ubuntu0.18.04.6 [5,092 kB]
Get:33 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libgs9 amd64 9.26~dfsg+0-0ubuntu0.18.04.6 [2,264 kB]
Get:34 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 ghostscript amd64 9.26~dfsg+0-0ubuntu0.18.04.6 [51.3 kB]
Get:35 http://archive.ubuntu.com/ubuntu bionic/main amd64 gsfonts all 1:8.11+urwcyr1.0.7~pre44-4.4 [3,120 kB]
Get:36 http://archive.ubuntu.com/ubuntu bionic/main amd64 hicolor-icon-theme all 0.17-2 [9,976 B]
Get:37 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 imagemagick-6.q16 amd64 8:6.9.7.4+dfsg-16ubuntu6.4 [423 kB]
Get:38 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 imagemagick amd64 8:6.9.7.4+dfsg-16ubuntu6.4 [14.2 kB]
Get:39 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpixman-1-0 amd64 0.34.0-2 [229 kB]
Get:40 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxcb-render0 amd64 1.13-1 [14.7 kB]
Get:41 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxcb-shm0 amd64 1.13-1 [5,572 B]
Get:42 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxrender1 amd64 1:0.9.10-1 [18.7 kB]
Get:43 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libcairo2 amd64 1.15.10-2ubuntu0.1 [580 kB]
Get:44 http://archive.ubuntu.com/ubuntu bionic/main amd64 libcupsfilters1 amd64 1.20.2-0ubuntu3 [108 kB]
Get:45 http://archive.ubuntu.com/ubuntu bionic/main amd64 libdatrie1 amd64 0.2.10-7 [17.8 kB]
Get:46 http://archive.ubuntu.com/ubuntu bionic/main amd64 libdjvulibre-text all 3.5.27.1-8 [49.1 kB]
Get:47 http://archive.ubuntu.com/ubuntu bionic/main amd64 libdjvulibre21 amd64 3.5.27.1-8 [559 kB]
Get:48 http://archive.ubuntu.com/ubuntu bionic/main amd64 libilmbase12 amd64 2.2.0-11ubuntu2 [71.4 kB]
Get:49 http://archive.ubuntu.com/ubuntu bionic/main amd64 libopenexr22 amd64 2.2.0-11.1ubuntu1 [560 kB]
Get:50 http://archive.ubuntu.com/ubuntu bionic/main amd64 libthai-data all 0.1.27-2 [133 kB]
Get:51 http://archive.ubuntu.com/ubuntu bionic/main amd64 libthai0 amd64 0.1.27-2 [18.0 kB]
Get:52 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpango-1.0-0 amd64 1.40.14-1ubuntu0.1 [153 kB]
Get:53 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpangoft2-1.0-0 amd64 1.40.14-1ubuntu0.1 [33.2 kB]
Get:54 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpangocairo-1.0-0 amd64 1.40.14-1ubuntu0.1 [20.8 kB]
Get:55 http://archive.ubuntu.com/ubuntu bionic/main amd64 libwmf0.2-7 amd64 0.2.8.4-12 [150 kB]
Get:56 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagickcore-6.q16-3-extra amd64 8:6.9.7.4+dfsg-16ubuntu6.4 [62.3 kB]
Get:57 http://archive.ubuntu.com/ubuntu bionic/main amd64 libnetpbm10 amd64 2:10.0-15.3build1 [58.0 kB]
Get:58 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpaper-utils amd64 1.1.24+nmu5ubuntu1 [8,170 B]
Get:59 http://archive.ubuntu.com/ubuntu bionic/main amd64 netpbm amd64 2:10.0-15.3build1 [1,017 kB]
Get:60 http://archive.ubuntu.com/ubuntu bionic/universe amd64 nscd amd64 2.27-3ubuntu1 [78.1 kB]
Preconfiguring packages ...
Fetched 23.7 MB in 1s (30.4 MB/s)
Selecting previously unselected package ucf.
(Reading database ... 21007 files and directories currently installed.)
Preparing to unpack .../00-ucf_3.0038_all.deb ...
Moving old data out of the way
Unpacking ucf (3.0038) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../01-fonts-dejavu-core_2.37-1_all.deb ...
Unpacking fonts-dejavu-core (2.37-1) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../02-fontconfig-config_2.12.6-0ubuntu2_all.deb ...
Unpacking fontconfig-config (2.12.6-0ubuntu2) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../03-libfontconfig1_2.12.6-0ubuntu2_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.12.6-0ubuntu2) ...
Selecting previously unselected package fontconfig.
Preparing to unpack .../04-fontconfig_2.12.6-0ubuntu2_amd64.deb ...
Unpacking fontconfig (2.12.6-0ubuntu2) ...
Selecting previously unselected package fonts-droid-fallback.
Preparing to unpack .../05-fonts-droid-fallback_1%3a6.0.1r16-1.1_all.deb ...
Unpacking fonts-droid-fallback (1:6.0.1r16-1.1) ...
Selecting previously unselected package libjpeg-turbo8:amd64.
Preparing to unpack .../06-libjpeg-turbo8_1.5.2-0ubuntu5.18.04.1_amd64.deb ...
Unpacking libjpeg-turbo8:amd64 (1.5.2-0ubuntu5.18.04.1) ...
Selecting previously unselected package libfftw3-double3:amd64.
Preparing to unpack .../07-libfftw3-double3_3.3.7-1_amd64.deb ...
Unpacking libfftw3-double3:amd64 (3.3.7-1) ...
Selecting previously unselected package libjbig0:amd64.
Preparing to unpack .../08-libjbig0_2.1-3.1build1_amd64.deb ...
Unpacking libjbig0:amd64 (2.1-3.1build1) ...
Selecting previously unselected package libjpeg8:amd64.
Preparing to unpack .../09-libjpeg8_8c-2ubuntu8_amd64.deb ...
Unpacking libjpeg8:amd64 (8c-2ubuntu8) ...
Selecting previously unselected package liblcms2-2:amd64.
Preparing to unpack .../10-liblcms2-2_2.9-1ubuntu0.1_amd64.deb ...
Unpacking liblcms2-2:amd64 (2.9-1ubuntu0.1) ...
Selecting previously unselected package liblqr-1-0:amd64.
Preparing to unpack .../11-liblqr-1-0_0.4.2-2.1_amd64.deb ...
Unpacking liblqr-1-0:amd64 (0.4.2-2.1) ...
Selecting previously unselected package libtiff5:amd64.
Preparing to unpack .../12-libtiff5_4.0.9-5ubuntu0.1_amd64.deb ...
Unpacking libtiff5:amd64 (4.0.9-5ubuntu0.1) ...
Selecting previously unselected package imagemagick-6-common.
Preparing to unpack .../13-imagemagick-6-common_8%3a6.9.7.4+dfsg-16ubuntu6.4_all.deb ...
Unpacking imagemagick-6-common (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package libmagickcore-6.q16-3:amd64.
Preparing to unpack .../14-libmagickcore-6.q16-3_8%3a6.9.7.4+dfsg-16ubuntu6.4_amd64.deb ...
Unpacking libmagickcore-6.q16-3:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package libmagickwand-6.q16-3:amd64.
Preparing to unpack .../15-libmagickwand-6.q16-3_8%3a6.9.7.4+dfsg-16ubuntu6.4_amd64.deb ...
Unpacking libmagickwand-6.q16-3:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package poppler-data.
Preparing to unpack .../16-poppler-data_0.4.8-2_all.deb ...
Unpacking poppler-data (0.4.8-2) ...
Selecting previously unselected package libapparmor1:amd64.
Preparing to unpack .../17-libapparmor1_2.12-4ubuntu5.1_amd64.deb ...
Unpacking libapparmor1:amd64 (2.12-4ubuntu5.1) ...
Selecting previously unselected package libdbus-1-3:amd64.
Preparing to unpack .../18-libdbus-1-3_1.12.2-1ubuntu1_amd64.deb ...
Unpacking libdbus-1-3:amd64 (1.12.2-1ubuntu1) ...
Selecting previously unselected package dbus.
Preparing to unpack .../19-dbus_1.12.2-1ubuntu1_amd64.deb ...
Unpacking dbus (1.12.2-1ubuntu1) ...
Selecting previously unselected package libcap2:amd64.
Preparing to unpack .../20-libcap2_1%3a2.25-1.2_amd64.deb ...
Unpacking libcap2:amd64 (1:2.25-1.2) ...
Selecting previously unselected package libidn11:amd64.
Preparing to unpack .../21-libidn11_1.33-2.1ubuntu1_amd64.deb ...
Unpacking libidn11:amd64 (1.33-2.1ubuntu1) ...
Selecting previously unselected package fonts-noto-mono.
Preparing to unpack .../22-fonts-noto-mono_20171026-2_all.deb ...
Unpacking fonts-noto-mono (20171026-2) ...
Selecting previously unselected package libavahi-common-data:amd64.
Preparing to unpack .../23-libavahi-common-data_0.7-3.1ubuntu1.2_amd64.deb ...
Unpacking libavahi-common-data:amd64 (0.7-3.1ubuntu1.2) ...
Selecting previously unselected package libavahi-common3:amd64.
Preparing to unpack .../24-libavahi-common3_0.7-3.1ubuntu1.2_amd64.deb ...
Unpacking libavahi-common3:amd64 (0.7-3.1ubuntu1.2) ...
Selecting previously unselected package libavahi-client3:amd64.
Preparing to unpack .../25-libavahi-client3_0.7-3.1ubuntu1.2_amd64.deb ...
Unpacking libavahi-client3:amd64 (0.7-3.1ubuntu1.2) ...
Selecting previously unselected package libcups2:amd64.
Preparing to unpack .../26-libcups2_2.2.7-1ubuntu2.3_amd64.deb ...
Unpacking libcups2:amd64 (2.2.7-1ubuntu2.3) ...
Selecting previously unselected package libcupsimage2:amd64.
Preparing to unpack .../27-libcupsimage2_2.2.7-1ubuntu2.3_amd64.deb ...
Unpacking libcupsimage2:amd64 (2.2.7-1ubuntu2.3) ...
Selecting previously unselected package libijs-0.35:amd64.
Preparing to unpack .../28-libijs-0.35_0.35-13_amd64.deb ...
Unpacking libijs-0.35:amd64 (0.35-13) ...
Selecting previously unselected package libjbig2dec0:amd64.
Preparing to unpack .../29-libjbig2dec0_0.13-6_amd64.deb ...
Unpacking libjbig2dec0:amd64 (0.13-6) ...
Selecting previously unselected package libpaper1:amd64.
Preparing to unpack .../30-libpaper1_1.1.24+nmu5ubuntu1_amd64.deb ...
Unpacking libpaper1:amd64 (1.1.24+nmu5ubuntu1) ...
Selecting previously unselected package libgs9-common.
Preparing to unpack .../31-libgs9-common_9.26~dfsg+0-0ubuntu0.18.04.6_all.deb ...
Unpacking libgs9-common (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Selecting previously unselected package libgs9:amd64.
Preparing to unpack .../32-libgs9_9.26~dfsg+0-0ubuntu0.18.04.6_amd64.deb ...
Unpacking libgs9:amd64 (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Selecting previously unselected package ghostscript.
Preparing to unpack .../33-ghostscript_9.26~dfsg+0-0ubuntu0.18.04.6_amd64.deb ...
Unpacking ghostscript (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Selecting previously unselected package gsfonts.
Preparing to unpack .../34-gsfonts_1%3a8.11+urwcyr1.0.7~pre44-4.4_all.deb ...
Unpacking gsfonts (1:8.11+urwcyr1.0.7~pre44-4.4) ...
Selecting previously unselected package hicolor-icon-theme.
Preparing to unpack .../35-hicolor-icon-theme_0.17-2_all.deb ...
Unpacking hicolor-icon-theme (0.17-2) ...
Selecting previously unselected package imagemagick-6.q16.
Preparing to unpack .../36-imagemagick-6.q16_8%3a6.9.7.4+dfsg-16ubuntu6.4_amd64.deb ...
Unpacking imagemagick-6.q16 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package imagemagick.
Preparing to unpack .../37-imagemagick_8%3a6.9.7.4+dfsg-16ubuntu6.4_amd64.deb ...
Unpacking imagemagick (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package libpixman-1-0:amd64.
Preparing to unpack .../38-libpixman-1-0_0.34.0-2_amd64.deb ...
Unpacking libpixman-1-0:amd64 (0.34.0-2) ...
Selecting previously unselected package libxcb-render0:amd64.
Preparing to unpack .../39-libxcb-render0_1.13-1_amd64.deb ...
Unpacking libxcb-render0:amd64 (1.13-1) ...
Selecting previously unselected package libxcb-shm0:amd64.
Preparing to unpack .../40-libxcb-shm0_1.13-1_amd64.deb ...
Unpacking libxcb-shm0:amd64 (1.13-1) ...
Selecting previously unselected package libxrender1:amd64.
Preparing to unpack .../41-libxrender1_1%3a0.9.10-1_amd64.deb ...
Unpacking libxrender1:amd64 (1:0.9.10-1) ...
Selecting previously unselected package libcairo2:amd64.
Preparing to unpack .../42-libcairo2_1.15.10-2ubuntu0.1_amd64.deb ...
Unpacking libcairo2:amd64 (1.15.10-2ubuntu0.1) ...
Selecting previously unselected package libcupsfilters1:amd64.
Preparing to unpack .../43-libcupsfilters1_1.20.2-0ubuntu3_amd64.deb ...
Unpacking libcupsfilters1:amd64 (1.20.2-0ubuntu3) ...
Selecting previously unselected package libdatrie1:amd64.
Preparing to unpack .../44-libdatrie1_0.2.10-7_amd64.deb ...
Unpacking libdatrie1:amd64 (0.2.10-7) ...
Selecting previously unselected package libdjvulibre-text.
Preparing to unpack .../45-libdjvulibre-text_3.5.27.1-8_all.deb ...
Unpacking libdjvulibre-text (3.5.27.1-8) ...
Selecting previously unselected package libdjvulibre21:amd64.
Preparing to unpack .../46-libdjvulibre21_3.5.27.1-8_amd64.deb ...
Unpacking libdjvulibre21:amd64 (3.5.27.1-8) ...
Selecting previously unselected package libilmbase12:amd64.
Preparing to unpack .../47-libilmbase12_2.2.0-11ubuntu2_amd64.deb ...
Unpacking libilmbase12:amd64 (2.2.0-11ubuntu2) ...
Selecting previously unselected package libopenexr22:amd64.
Preparing to unpack .../48-libopenexr22_2.2.0-11.1ubuntu1_amd64.deb ...
Unpacking libopenexr22:amd64 (2.2.0-11.1ubuntu1) ...
Selecting previously unselected package libthai-data.
Preparing to unpack .../49-libthai-data_0.1.27-2_all.deb ...
Unpacking libthai-data (0.1.27-2) ...
Selecting previously unselected package libthai0:amd64.
Preparing to unpack .../50-libthai0_0.1.27-2_amd64.deb ...
Unpacking libthai0:amd64 (0.1.27-2) ...
Selecting previously unselected package libpango-1.0-0:amd64.
Preparing to unpack .../51-libpango-1.0-0_1.40.14-1ubuntu0.1_amd64.deb ...
Unpacking libpango-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Selecting previously unselected package libpangoft2-1.0-0:amd64.
Preparing to unpack .../52-libpangoft2-1.0-0_1.40.14-1ubuntu0.1_amd64.deb ...
Unpacking libpangoft2-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Selecting previously unselected package libpangocairo-1.0-0:amd64.
Preparing to unpack .../53-libpangocairo-1.0-0_1.40.14-1ubuntu0.1_amd64.deb ...
Unpacking libpangocairo-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Selecting previously unselected package libwmf0.2-7:amd64.
Preparing to unpack .../54-libwmf0.2-7_0.2.8.4-12_amd64.deb ...
Unpacking libwmf0.2-7:amd64 (0.2.8.4-12) ...
Selecting previously unselected package libmagickcore-6.q16-3-extra:amd64.
Preparing to unpack .../55-libmagickcore-6.q16-3-extra_8%3a6.9.7.4+dfsg-16ubuntu6.4_amd64.deb ...
Unpacking libmagickcore-6.q16-3-extra:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Selecting previously unselected package libnetpbm10.
Preparing to unpack .../56-libnetpbm10_2%3a10.0-15.3build1_amd64.deb ...
Unpacking libnetpbm10 (2:10.0-15.3build1) ...
Selecting previously unselected package libpaper-utils.
Preparing to unpack .../57-libpaper-utils_1.1.24+nmu5ubuntu1_amd64.deb ...
Unpacking libpaper-utils (1.1.24+nmu5ubuntu1) ...
Selecting previously unselected package netpbm.
Preparing to unpack .../58-netpbm_2%3a10.0-15.3build1_amd64.deb ...
Unpacking netpbm (2:10.0-15.3build1) ...
Selecting previously unselected package nscd.
Preparing to unpack .../59-nscd_2.27-3ubuntu1_amd64.deb ...
Unpacking nscd (2.27-3ubuntu1) ...
Setting up libgs9-common (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Setting up imagemagick-6-common (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Setting up fonts-droid-fallback (1:6.0.1r16-1.1) ...
Setting up libxcb-render0:amd64 (1.13-1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Setting up gsfonts (1:8.11+urwcyr1.0.7~pre44-4.4) ...
Setting up liblcms2-2:amd64 (2.9-1ubuntu0.1) ...
Setting up libjbig0:amd64 (2.1-3.1build1) ...
Setting up fonts-dejavu-core (2.37-1) ...
Setting up poppler-data (0.4.8-2) ...
Setting up libdjvulibre-text (3.5.27.1-8) ...
Setting up libnetpbm10 (2:10.0-15.3build1) ...
Setting up libcap2:amd64 (1:2.25-1.2) ...
Setting up libdatrie1:amd64 (0.2.10-7) ...
Setting up libjpeg-turbo8:amd64 (1.5.2-0ubuntu5.18.04.1) ...
Setting up libfftw3-double3:amd64 (3.3.7-1) ...
Setting up ucf (3.0038) ...
Setting up fonts-noto-mono (20171026-2) ...
Setting up libilmbase12:amd64 (2.2.0-11ubuntu2) ...
Setting up liblqr-1-0:amd64 (0.4.2-2.1) ...
Setting up libjbig2dec0:amd64 (0.13-6) ...
Setting up libpixman-1-0:amd64 (0.34.0-2) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Setting up libapparmor1:amd64 (2.12-4ubuntu5.1) ...
Setting up libijs-0.35:amd64 (0.35-13) ...
Setting up libxcb-shm0:amd64 (1.13-1) ...
Setting up libxrender1:amd64 (1:0.9.10-1) ...
Setting up libthai-data (0.1.27-2) ...
Setting up hicolor-icon-theme (0.17-2) ...
Setting up libidn11:amd64 (1.33-2.1ubuntu1) ...
Setting up libdbus-1-3:amd64 (1.12.2-1ubuntu1) ...
Setting up libavahi-common-data:amd64 (0.7-3.1ubuntu1.2) ...
Setting up libjpeg8:amd64 (8c-2ubuntu8) ...
Setting up libpaper1:amd64 (1.1.24+nmu5ubuntu1) ...

Creating config file /etc/papersize with new version
Setting up libpaper-utils (1.1.24+nmu5ubuntu1) ...
Setting up fontconfig-config (2.12.6-0ubuntu2) ...
Setting up nscd (2.27-3ubuntu1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Setting up libdjvulibre21:amd64 (3.5.27.1-8) ...
Setting up libtiff5:amd64 (4.0.9-5ubuntu0.1) ...
Setting up libthai0:amd64 (0.1.27-2) ...
Setting up libopenexr22:amd64 (2.2.0-11.1ubuntu1) ...
Setting up netpbm (2:10.0-15.3build1) ...
Setting up libavahi-common3:amd64 (0.7-3.1ubuntu1.2) ...
Setting up dbus (1.12.2-1ubuntu1) ...
Setting up libwmf0.2-7:amd64 (0.2.8.4-12) ...
Setting up libfontconfig1:amd64 (2.12.6-0ubuntu2) ...
Setting up libmagickcore-6.q16-3:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Setting up libmagickwand-6.q16-3:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Setting up libavahi-client3:amd64 (0.7-3.1ubuntu1.2) ...
Setting up fontconfig (2.12.6-0ubuntu2) ...
Regenerating fonts cache... done.
Setting up libcups2:amd64 (2.2.7-1ubuntu2.3) ...
Setting up libcairo2:amd64 (1.15.10-2ubuntu0.1) ...
Setting up imagemagick-6.q16 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
update-alternatives: using /usr/bin/compare-im6.q16 to provide /usr/bin/compare (compare) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/compare.1.gz because associated file /usr/share/man/man1/compare-im6.q16.1.gz (of link group compare) doesn't exist
update-alternatives: using /usr/bin/compare-im6.q16 to provide /usr/bin/compare-im6 (compare-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/compare-im6.1.gz because associated file /usr/share/man/man1/compare-im6.q16.1.gz (of link group compare-im6) doesn't exist
update-alternatives: using /usr/bin/animate-im6.q16 to provide /usr/bin/animate (animate) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/animate.1.gz because associated file /usr/share/man/man1/animate-im6.q16.1.gz (of link group animate) doesn't exist
update-alternatives: using /usr/bin/animate-im6.q16 to provide /usr/bin/animate-im6 (animate-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/animate-im6.1.gz because associated file /usr/share/man/man1/animate-im6.q16.1.gz (of link group animate-im6) doesn't exist
update-alternatives: using /usr/bin/convert-im6.q16 to provide /usr/bin/convert (convert) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/convert.1.gz because associated file /usr/share/man/man1/convert-im6.q16.1.gz (of link group convert) doesn't exist
update-alternatives: using /usr/bin/convert-im6.q16 to provide /usr/bin/convert-im6 (convert-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/convert-im6.1.gz because associated file /usr/share/man/man1/convert-im6.q16.1.gz (of link group convert-im6) doesn't exist
update-alternatives: using /usr/bin/composite-im6.q16 to provide /usr/bin/composite (composite) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/composite.1.gz because associated file /usr/share/man/man1/composite-im6.q16.1.gz (of link group composite) doesn't exist
update-alternatives: using /usr/bin/composite-im6.q16 to provide /usr/bin/composite-im6 (composite-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/composite-im6.1.gz because associated file /usr/share/man/man1/composite-im6.q16.1.gz (of link group composite-im6) doesn't exist
update-alternatives: using /usr/bin/conjure-im6.q16 to provide /usr/bin/conjure (conjure) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/conjure.1.gz because associated file /usr/share/man/man1/conjure-im6.q16.1.gz (of link group conjure) doesn't exist
update-alternatives: using /usr/bin/conjure-im6.q16 to provide /usr/bin/conjure-im6 (conjure-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/conjure-im6.1.gz because associated file /usr/share/man/man1/conjure-im6.q16.1.gz (of link group conjure-im6) doesn't exist
update-alternatives: using /usr/bin/import-im6.q16 to provide /usr/bin/import (import) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/import.1.gz because associated file /usr/share/man/man1/import-im6.q16.1.gz (of link group import) doesn't exist
update-alternatives: using /usr/bin/import-im6.q16 to provide /usr/bin/import-im6 (import-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/import-im6.1.gz because associated file /usr/share/man/man1/import-im6.q16.1.gz (of link group import-im6) doesn't exist
update-alternatives: using /usr/bin/identify-im6.q16 to provide /usr/bin/identify (identify) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/identify.1.gz because associated file /usr/share/man/man1/identify-im6.q16.1.gz (of link group identify) doesn't exist
update-alternatives: using /usr/bin/identify-im6.q16 to provide /usr/bin/identify-im6 (identify-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/identify-im6.1.gz because associated file /usr/share/man/man1/identify-im6.q16.1.gz (of link group identify-im6) doesn't exist
update-alternatives: using /usr/bin/stream-im6.q16 to provide /usr/bin/stream (stream) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/stream.1.gz because associated file /usr/share/man/man1/stream-im6.q16.1.gz (of link group stream) doesn't exist
update-alternatives: using /usr/bin/stream-im6.q16 to provide /usr/bin/stream-im6 (stream-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/stream-im6.1.gz because associated file /usr/share/man/man1/stream-im6.q16.1.gz (of link group stream-im6) doesn't exist
update-alternatives: using /usr/bin/display-im6.q16 to provide /usr/bin/display (display) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/display.1.gz because associated file /usr/share/man/man1/display-im6.q16.1.gz (of link group display) doesn't exist
update-alternatives: using /usr/bin/display-im6.q16 to provide /usr/bin/display-im6 (display-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/display-im6.1.gz because associated file /usr/share/man/man1/display-im6.q16.1.gz (of link group display-im6) doesn't exist
update-alternatives: using /usr/bin/montage-im6.q16 to provide /usr/bin/montage (montage) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/montage.1.gz because associated file /usr/share/man/man1/montage-im6.q16.1.gz (of link group montage) doesn't exist
update-alternatives: using /usr/bin/montage-im6.q16 to provide /usr/bin/montage-im6 (montage-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/montage-im6.1.gz because associated file /usr/share/man/man1/montage-im6.q16.1.gz (of link group montage-im6) doesn't exist
update-alternatives: using /usr/bin/mogrify-im6.q16 to provide /usr/bin/mogrify (mogrify) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/mogrify.1.gz because associated file /usr/share/man/man1/mogrify-im6.q16.1.gz (of link group mogrify) doesn't exist
update-alternatives: using /usr/bin/mogrify-im6.q16 to provide /usr/bin/mogrify-im6 (mogrify-im6) in auto mode
update-alternatives: warning: skip creation of /usr/share/man/man1/mogrify-im6.1.gz because associated file /usr/share/man/man1/mogrify-im6.q16.1.gz (of link group mogrify-im6) doesn't exist
Setting up libcupsfilters1:amd64 (1.20.2-0ubuntu3) ...
Setting up libcupsimage2:amd64 (2.2.7-1ubuntu2.3) ...
Setting up libpango-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Setting up imagemagick (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Setting up libgs9:amd64 (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Setting up ghostscript (9.26~dfsg+0-0ubuntu0.18.04.6) ...
Setting up libpangoft2-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Setting up libpangocairo-1.0-0:amd64 (1.40.14-1ubuntu0.1) ...
Setting up libmagickcore-6.q16-3-extra:amd64 (8:6.9.7.4+dfsg-16ubuntu6.4) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
npm WARN deprecated boom@4.3.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
npm WARN deprecated domelementtype@1.3.0: update to domelementtype@1.3.1
npm WARN deprecated istanbul@0.4.5: This module is no longer maintained, try this instead:
npm WARN deprecated   npm i nyc
npm WARN deprecated Visit https://istanbul.js.org/integrations for other alternatives.
npm WARN deprecated kad-fs@0.0.4: Please upgrade to @kadenceproject/kadence - See https://kadence.github.io
npm WARN deprecated kad-memstore@0.0.1: Please upgrade to @kadenceproject/kadence - See https://kadence.github.io
npm WARN deprecated nsp@3.2.1: The Node Security Platform service is shutting down 9/30 - https://blog.npmjs.org/post/175511531085/the-node-security-platform-service-is-shutting
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
/usr/bin/mwoffliner -> /usr/lib/node_modules/mwoffliner/bin/mwoffliner.script.js
/usr/bin/mwmatrixoffliner -> /usr/lib/node_modules/mwoffliner/bin/mwmatrixoffliner.script.js

> dtrace-provider@0.8.7 install /usr/lib/node_modules/mwoffliner/node_modules/dtrace-provider
> node-gyp rebuild || node suppress-error.js

gyp WARN EACCES user "undefined" does not have permission to access the dev dir "/root/.node-gyp/10.15.1"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/mwoffliner/node_modules/dtrace-provider/.node-gyp"
gyp WARN install got an error, rolling back install
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/mwoffliner/node_modules/dtrace-provider/.node-gyp'
gyp ERR! System Linux 4.15.0-43-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/lib/node_modules/mwoffliner/node_modules/dtrace-provider
gyp ERR! node -v v10.15.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 

> @openzim/libzim@2.0.7 install /usr/lib/node_modules/mwoffliner/node_modules/@openzim/libzim
> npm run download && (npm run build:native || echo Warning: problems with native build)

npm ERR! path /root/.npm/_logs
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall scandir
npm ERR! Error: EACCES: permission denied, scandir '/root/.npm/_logs'
npm ERR!  { [Error: EACCES: permission denied, scandir '/root/.npm/_logs']
npm ERR!   stack:
npm ERR!    'Error: EACCES: permission denied, scandir \'/root/.npm/_logs\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'scandir',
npm ERR!   path: '/root/.npm/_logs' }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator (though this is not recommended).
glob error { [Error: EACCES: permission denied, scandir '/root/.npm/_logs']
  stack:
   'Error: EACCES: permission denied, scandir \'/root/.npm/_logs\'',
  errno: -13,
  code: 'EACCES',
  syscall: 'scandir',
  path: '/root/.npm/_logs' }

> @openzim/libzim@2.0.7 download /usr/lib/node_modules/mwoffliner/node_modules/@openzim/libzim
> node ./download-libzim.js

/usr/lib/node_modules/mwoffliner/node_modules/mkdirp/index.js:90
                    throw err0;
                    ^

Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/mwoffliner/node_modules/@openzim/libzim/download'
    at Object.mkdirSync (fs.js:753:3)
    at Function.sync (/usr/lib/node_modules/mwoffliner/node_modules/mkdirp/index.js:71:13)
    at Object.<anonymous> (/usr/lib/node_modules/mwoffliner/node_modules/@openzim/libzim/download-libzim.js:9:8)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:283:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @openzim/libzim@2.0.7 download: `node ./download-libzim.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @openzim/libzim@2.0.7 download script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! code ELIFECYCLE
npm ERR! errno 243
npm ERR! @openzim/libzim@2.0.7 install: `npm run download && (npm run build:native || echo Warning: problems with native build)`
npm ERR! Exit status 243
npm ERR! 
npm ERR! Failed at the @openzim/libzim@2.0.7 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-02-24T11_32_02_027Z-debug.log
```

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,18 +1,14 @@
-FROM openzim/zimwriterfs:latest
+FROM node:10
 
 # Basics
 RUN apt-get update && apt-get install -y make g++ curl git
 
-
-# Install npm & nodejs
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash && \
-    apt-get install --yes nodejs
-
 # Install mwoffliner
-RUN apt-get install -y nscd file imagemagick && \
-    npm install -g mwoffliner && \
-    apt-get remove -y make g++
+RUN apt-get install -y imagemagick
+RUN npm --global config set user root
+RUN npm install -g openzim/mwoffliner
+RUN apt-get remove -y make g++
 
 # Boot commands
-# COPY .custom-bashrc /root/
+COPY .custom-bashrc /root/
 CMD mwoffliner

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -6,7 +6,7 @@ RUN apt-get update && apt-get install -y make g++ curl git
 # Install mwoffliner
 RUN apt-get install -y imagemagick
 RUN npm --global config set user root
-RUN npm install -g openzim/mwoffliner
+RUN npm install -g mwoffliner
 RUN apt-get remove -y make g++
 
 # Boot commands

